src/css/application.css

```diff
@@ -329,14 +329,32 @@ body {
   flex: 1 1 auto;
   display: flex;
   flex-direction: row;
+  height: 100vh;
 }
 
 .environment-column {
   flex: 1 1 50%;
 }
 
+.container-label {
+  position: absolute;
+  bottom: 1em;
+  right: 1em;
+  background-color: black;
+  color: white;
+  z-index: 1;
+  padding: 0.2rem;
+  opacity: 0.2;
+  font-size: 0.8rem;
```

Comment:
We should also specify an explicit font family here—right now the editor labels are inheriting a different font from the preview label, which is making them appear to be a different size.